packages/next-swc/crates/next-core/src/next_font/google/mod.rs

```diff
@@ -169,11 +168,14 @@ impl ImportMappingReplacement for NextFontGoogleCssModuleReplacer {
         let options = font_options_from_query_map(*query_vc, font_data);
         let stylesheet_url = get_stylesheet_url_from_options(options, font_data);
         let request_hash = get_request_hash(*query_vc);
-        let scoped_font_family =
-            get_scoped_font_family(FontFamilyType::WebFont.cell(), options, request_hash);
+        let scoped_font_family = get_scoped_font_family(
+            FontFamilyType::WebFont.cell(),
+            options.await?.font_family.to_owned(),
+            request_hash,
+        );
         let css_virtual_path = next_js_file_path("internal/font/google").join(&format!(
             "/{}.module.css",
-            get_request_id(*query_vc, font_data).await?
+            get_request_id(options.await?.font_family.clone(), request_hash).await?
```

Comment:
Can we pass the options into the function? That way the outer `result` doesn't depend on the value of options.

Reply:
Need to use the new Vc method?